lib/mysql/index.js

```diff
@@ -49,6 +49,11 @@ const selectAllFrom = (mysql, table) =>
 const insert = Query.insert
 
 
+const update = R.curry((mysql, table, id, params) =>
```

Comment:
actually, I just changed the name of the test file to match the file being tested.

test/lib/mysql/index.spec.js should exist now